### PR TITLE
Bump Tweaked to v1.6

### PR DESCRIPTION
**Pull request by calvarado194**
Changelog:
- Build is now compiled on cpp17, allowing it to run on more Linux distros (most notably, it runs on Steam Deck).
- Defaults to 60fps and vsync on.
- Removed Courier New from the font list to avoid license violation.
- Controller axis (such as analog sticks) can now be mapped
- Currently playing song is no longer restarted if you leave the Soundtrack Select menu without changing your soundtrack choice
- Accept button doesn't advance menu options anymore
- Game can be closed when unfocused
- Unfocusing the game and refocusing won't make the game restart music playback if music volume is set to 0%
- Pressing the Map button with the map now open closes the map
- Gamepad mapping now has defaults based on SDL game controller API.
- Gamepad remap menu now uses icons instead of generic names.
- Monogram font has been updated to v1.1
- In-game mouse cursor is now hidden after 10 minutes of inactivity
- Arrows in the pause menu are now always drawn
- Tweaked now follows XDG directory specification by default to store Config files, Tweak data and Save data. To enable freeware behavior, you can provide a portable.txt file.
- Fixed Original Minimap not displaying Quote's position correctly under Mirror Mode
- Pressing the Pause button with the minimap open opens the actual Pause Menu rather than the vanilla ESC menu
- Updated credits sequence to credit the mod developers
- Bundled manual has been updated with a new look, a list of the default keyboard and joystick bindings, and other changes.
- Updated licence.txt file.

For CSR I've already included an empty `portable.txt` to avoid save data issues.

### CODE REVIEW

**Comment by Miepee** [comment]
> Build is now compiled on cpp17, allowing it to run on more Linux distros (most notably, it runs on Steam Deck).

It running on steam deck is not because of cpp17, but because we're linking against a much older GLIBC.


> In-game mouse cursor is now hidden after 10 minutes of inactivity

\* 10 Seconds, not minutes.

> Tweaked now follows XDG directory specification by default to store Config files, Tweak data and Save data. 

If this changelog is posted anywhere publically, this might be worth rephrasing a little, because the XDG spec only really makes sense in the context of linux / non-apple unix.

File: pre_edited_cs/tweaked/error.log [hunk 1]
```diff
@@ -0,0 +1 @@
+/media/ema/61B0470F323D03A2/dev/cave-story-randomizer/pre_edited_cs/tweaked/data/buttons.bmp,1
```
Comment (duncathan) [inline pre_edited_cs/tweaked/error.log:1]: oops

File: pre_edited_cs/tweaked/data/casts.png
Comment (duncathan) [inline pre_edited_cs/tweaked/data/casts.png:1]: is it intended that your casts.png files don't include the icons for the inventory display at the end of the credits?

**Review by duncathan — CHANGES_REQUESTED** [review]

File: pre_edited_cs/tweaked/data/casts.png
Comment (calvarado194) [reply pre_edited_cs/tweaked/data/casts.png:1]: oops, my bad, added them back

**Review by duncathan — APPROVED** [review]

**Pull request merged**